model/host/hostutil.go

```diff
@@ -383,16 +383,37 @@ func (h *Host) buildLocalJasperClientRequest(config evergreen.HostJasperConfig,
 }
 
 // WriteJasperCredentialsFileCommand builds the command to write the Jasper
-// credentials to a file.
-func (h *Host) WriteJasperCredentialsFileCommand(creds *rpc.Credentials) (string, error) {
+// credentials and Splunk credentials to files.
+func (h *Host) WriteJasperCredentialsFilesCommands(settings *evergreen.Settings, creds *rpc.Credentials) (string, error) {
```

Comment:
I think it would be cleaner to pass a SplunkConnectionInfo object rather than the entire settings here

Reply:
Done.